cypress-io/cypress
# chore: manually fix npm package changelogs

- Closes #24112

### User facing changelog


N/A

### Additional details


There was an issue after we switched from releasing NPM packages from `master` to `develop` where some packages had existing tags that were pointing to commits that didn't exist on `develop`. This caused those packages not to be published to NPM and their changelogs were getting updated with duplicate entries every time the `npm-release` step ran.

I fixed the tags issue by deleting the tags that were pointing to the commit on `master`, and then re-tagging the corresponding commit (usually a merge commit) on `develop`. Then when `npm-release` ran again, it published some new versions of some of the packages who's code had been update since the issue began. These packages should be bundled with the next version of Cypress.

This PR manually fixes each of the affected changelogs so that they match what we've actually released.

I added an issue to make it so that the changelog doesn't get updated if the release fails for the package https://github.com/cypress-io/cypress/issues/24247

### Steps to test


Check each of the command logs and verify that they are correct and that the latest entry matches the version on NPM for each package.

You can also see the job where the new versions of the package were released successfully here
https://app.circleci.com/pipelines/github/cypress-io/cypress/44696/workflows/33ef3ef4-1728-450e-80ec-f0f2a06c7cdc/jobs/1874815

### How has the user experience changed?


N/A

### PR Tasks


- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
